src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -45,152 +48,297 @@
 import seedu.address.model.tag.Tag;
 import seedu.address.testutil.PatientBuilder;
 
+
 public class AddCommandParserTest {
-    private AddCommandParser parser = new AddCommandParser();
+    private final AddCommandParser parser = new AddCommandParser();
+
+
+    @Test
+    public void getPrefixesPresent_namePrefixPresent_returnedCorrectPrefixList() {
```

Comment:
Not sure whether this is trivial but it might be nice to add a testcase for an empty input (No prefixes present)